scripts/release/index.js

```diff
@@ -38,6 +38,7 @@ cooker.cook('publish', [
       cook.pushTagToRemote,
       cook.createReleaseNotes(releaseNotesTemplate),
       cook.createGithubRelease,
+      cook.runNpmScript('deploy'),
```

Comment:
cc @Guria , got any comments on this? Not sure how far I will get offline and do not understand how this script ever worked without having the dependencies?